src/main/java/emissary/config/ConfigUtil.java

```diff
@@ -104,16 +103,16 @@ public class ConfigUtil {
      * XML bean configuration with Jetty 6 does a poor job of propagating exceptions during configure. Once we update to
      * Jetty 9, evaluate removing this. Otherwise, there is no way I could find to stop the server from starting if there is
      * a configuration error.
-     * 
+     * <p>
      * TODO: evaluate whether we can change this now that we are on jetty 9
```

Comment:
Has this evaluation been done? Or is this still an open task? If so we should open an issue for it.

Reply:
https://github.com/NationalSecurityAgency/burrito-grande/issues/1919